<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rbTerm">
        <property name="text">
         <string>Term</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbCumulative">
        <property name="text">
         <string>Cumulative</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbVIP">
        <property name="text">
         <string>VIP</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Add coefficient:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="eAddCoef"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Cliant name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="eClientName"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Money:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="eStartMoney"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Coefficient</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="eCoefficient"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Max month</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="eMaxMonth"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pbtAdd">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbtNextDay">
          <property name="text">
           <string>Next</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>Cliant Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Coefficient</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Monthes</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Start money</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Money</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Month</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>rbVIP</sender>
   <signal>clicked()</signal>
   <receiver>eAddCoef</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbVIP</sender>
   <signal>clicked()</signal>
   <receiver>label_3</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>339</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbCumulative</sender>
   <signal>clicked()</signal>
   <receiver>eAddCoef</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbCumulative</sender>
   <signal>clicked()</signal>
   <receiver>label_3</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbTerm</sender>
   <signal>clicked()</signal>
   <receiver>eAddCoef</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>181</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbTerm</sender>
   <signal>clicked()</signal>
   <receiver>label_3</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>94</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
